Tech-Talk-LA/TechTalk
# User Feed set-up

## Description
- Navigation routing from login page to user feed page was established and a simple mock-up of the modularized recommended user feed was established

## What I changed
- App.js: Added screen for User Feed view to navigation container
- UserFeed.js: added file and created the view for the page with link to the Feed.js file
- Feed.js: added file and established render function for passing state data for individual recommended user boxes to the FeedBox.js file
- FeedBox.js: added file and outlined a simple view of the required user data for the recommended user feed

## How to test
- In terminal: expo start
- Once bundling is completed, navigate to web browser either via Expo GUI in browser or type w into the terminal
- Should open browser page to Login view. Navigate to User Feed Page by clicking login button.
